pajbot/apiwrappers/queup.py

```diff
@@ -92,8 +92,7 @@ def fetch_current_song(self, room_id):
             if e.response.status_code == 404:
                 # No songs in active queue.
                 return None
-            else:
-                raise e
+            raise e
```

Comment:
Let's add a little space
```suggestion

            raise e
```